sdk/cosmos/azure-cosmos/src/main/java/module-info.java

```diff
@@ -35,10 +35,12 @@
 
     // exporting implementation packages specifically for cosmos encryption
     exports com.azure.cosmos.implementation to com.azure.cosmos.encryption;
-    exports com.azure.cosmos.implementation.query to com.azure.cosmos.encryption;
-    exports com.azure.cosmos.implementation.patch to com.azure.cosmos.encryption;
     exports com.azure.cosmos.implementation.batch to com.azure.cosmos.encryption;
     exports com.azure.cosmos.implementation.caches to com.azure.cosmos.encryption;
+    exports com.azure.cosmos.implementation.feedranges to com.azure.cosmos.encryption;
```

Comment:
@kushagraThapar @FabianMeiswinkel this was alphabetical re-ordering and the changes to using modules in testing found that `azure-cosmos` didn't export `feedranges` to `azure-cosmos-encryption`